examples/databricks_DBRX_website_bot/main.py

```diff
@@ -17,10 +20,17 @@ def build_RAG(
     embed_model="mixedbread-ai/mxbai-embed-large-v1",
     uri="~/tmp/lancedb_hogwart",
     force_create_embeddings=False,
+    illustrate=True,
+    diffuser_model="sdxl",
 ):
     Settings.embed_model = HuggingFaceEmbedding(model_name=embed_model)
     Settings.llm = Databricks(model="databricks-dbrx-instruct")
-
+    if illustrate:
+        print("Loading sdxl model")
+        model, steps = load_models(diffuser_model)
+        # This is a hack to tradeoff between speed and quality
+        steps = 1
```

Comment:
If you're getting good enough speeds, Remove this line if you want to improve image quality
@changhiskhan